mixer/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM scratch
+
+# obtained from debian ca-certs deb using fetch_cacerts.sh
+ADD ca-certificates.tgz /
```

Comment:
is that a standard go feature or we have code to do something to this tgz ?

for fortio I only need `/etc/ssl/certs/ca-certificates.crt` - what else is in that tgz ?

https://github.com/istio/fortio/blob/master/Dockerfile#L14

See also from that example how you can do multi stage docker build to copy the output of one into the next to create clean `FROM scratch` while controlling the build env